### PR TITLE
Update dependency versions and build script

### PR DESCRIPTION
**Pull request by mrichards42**
Based on changes I had to make for `crm-works`

### CODE REVIEW

File: resources/leiningen/new/kraken_works/project.clj [hunk 1]
```diff
@@ -3,21 +3,20 @@
...
-                 [com.datomic/datomic-pro "0.9.5561"
+                 [com.datomic/datomic-pro "0.9.5661"
                   :exclusions [org.slf4j/slf4j-nop
                                org.slf4j/slf4j-log4j12]]
-                 [com.amazonaws/aws-java-sdk-dynamodb "1.11.128"
+                 [com.amazonaws/aws-java-sdk-dynamodb "1.11.261"
                   :exclusions [commons-codec commons-logging]]
                  [ch.qos.logback/logback-classic "1.2.2"]
-                 [democracyworks/kehaar "0.11.2"]]
+                 [democracyworks/kehaar "0.11.3"]]
```
Comment (cap10morgan) [inline resources/leiningen/new/kraken_works/project.clj]: Version 0.11.4 of kehaar was released on Jan. 26. Can we make sure all these deps are the latest (compatible) releases?

**Review by cap10morgan — CHANGES_REQUESTED** [review]
Thanks for doing this! It's very helpful but easy to neglect.

Just one thing to look into w/r/t dep versions.

**Comment by mrichards42** [comment]
Everything's up to date now according to `lein ancient`. Looks like the datomic changes are [minor](https://docs.datomic.com/on-prem/changes.html) as well.

**Comment by mrichards42** [comment]
Some of the deps are out of date, and some of the changes are no longer necessary, so I'll close and open a new PR

**Pull request closed without merging**